lib/Dancer2/CLI/Gen.pm

```diff
@@ -220,10 +217,10 @@ sub _copy_templates {
         }
 
         if( $from !~ m/\.(ico|jpg|png|css|eot|map|swp|ttf|svg|woff|woff2|js)$/ ) {
-            $content = _process_template($content, $vars);
+            $content = $self->_process_template($content, $vars);
         }
 
-        open( my $fh, '>:raw', $to ) or die "unable to open file `$to' for writing: $!";
+        open( my $fh, '>:raw', $to ) or die "unable to open file `$to` for writing: $!";
         print $fh $content;
         close $fh;
```

Comment:
and on line 228 (GH not allowing me to add a comment there):
```
$t0->chmod("0755") or ...
```

Reply:
Done.